vendor/github.com/containers/common/pkg/apparmor/apparmor_linux.go

```diff
@@ -11,7 +11,7 @@ import (
 	"path"
 	"strconv"
 	"strings"
-	"text/template"
+	"html/template"
```

Comment:
we shouldn't change this here. It should be changed in the source and revendored

Reply:
where is the source @haircommander ? thanks

Reply:
@haircommander so i changed in template and run `make vendor`

Reply:
you can find it based on where the vendor directory is:
github.com/containers/common/tree/master/pkg/apparmor/apparmor_linux.go

Reply:
thanks @haircommander so I should make the PR in that repo and close this one? thanks

Reply:
yes I would recommend that, thanks :)

Reply:
ok thanks @haircommander